### PR TITLE
Feature/dev 028

### PR DESCRIPTION
**Pull request by Enrickzz**
-added app client and app service
-implemented get it

### CODE REVIEW

**Comment by raelsj** [comment]
LINTS!!!!!!!!!!!

File: lib/feature/home_page/widgets/pokemon_tile.dart [hunk 1]
```diff
@@ -1,13 +1,11 @@
-import 'dart:convert';
+import 'package:pokedex_async_redux/api/api_service.dart';
+import 'package:pokedex_async_redux/api/models/details_pokemon_model.dart';
 import 'package:pokedex_async_redux/api/models/pokemon_model.dart';
-import 'package:pokedex_async_redux/api/models/pokemon_type_model.dart';
-import 'package:pokedex_async_redux/api/models/specific_type_model.dart';
-import 'package:pokedex_async_redux/api/models/sub_type_model.dart';
 import 'package:pokedex_async_redux/feature/pokemon_details_page.dart/pokemon_details_page_connector.dart';
 import 'package:pokedex_async_redux/feature/widgets/pill_container_widget.dart';
+import 'package:pokedex_async_redux/utilities/app_starter.dart';
```
Comment (dbenedictos) [inline lib/feature/home_page/widgets/pokemon_tile.dart]: unused import

File: lib/state/actions/filter_actions.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import 'package:pokedex_async_redux/api/apI_service.dart';
```
Comment (dbenedictos) [inline lib/state/actions/filter_actions.dart]: update the import

File: lib/state/actions/home_page_actions.dart [hunk 3]
```diff
@@ -1,52 +1,17 @@
 import 'dart:async';
-import 'package:pokedex_async_redux/api/handlers/details_pokemon_handler.dart';
-import 'package:pokedex_async_redux/api/handlers/evolution_handler.dart';
-import 'package:pokedex_async_redux/api/handlers/pokemon_handler.dart';
+import 'package:pokedex_async_redux/api/apI_service.dart';
```
Comment (dbenedictos) [inline lib/state/actions/home_page_actions.dart]: and this

File: lib/state/actions/pokemon_details_actions.dart [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import 'package:pokedex_async_redux/api/apI_service.dart';
```
Comment (dbenedictos) [inline lib/state/actions/pokemon_details_actions.dart]: and this

**Review by dbenedictos — APPROVED** [review]

File: lib/api/handlers/details_pokemon_handler.dart [hunk 5]
```diff
@@ -1,40 +1,38 @@
...
-      List movesMap = endResponse['moves'];
-      List abilitiesMap = endResponse['abilities'];
-      var speciesMap = endResponse['species'];
+  DetailsPokemonHandler({required this.apiClient});
+  final ApiClient apiClient;
+
+  Future<DetailsPokemon?> getDetails(String id) async {
+    final baseUri = Uri.parse(apiClient.dio.options.baseUrl);
+    final uri = baseUri.replace(path: '${baseUri.path}$pokemonsPath$id');
+    return apiClient.dio.getUri(uri).then((responsee) {
```
Comment (raelsj) [inline lib/api/handlers/details_pokemon_handler.dart:21]: responsee

**Pull request merged**